docs/consume-packages/Package-References-in-Project-Files.md

```diff
@@ -170,6 +170,20 @@ Conditions can also be applied at the `ItemGroup` level and will apply to all ch
 </ItemGroup>
 ```
 
+When you have many targets, it may be better to match ranges of TFMs, in which case you can use `IsTargetFrameworkCompatible`:
```

Comment:
Can we link to https://learn.microsoft.com/en-us/visualstudio/msbuild/property-functions?view=vs-2022#msbuild-targetframework-and-targetplatform-functions as well?
https://github.com/NuGet/docs.microsoft.com-nuget?tab=readme-ov-file#links

It's worth noting that the more complicated the conditions are, the more challenging it is for any tooling to update these correctly. 
I'd consider these to be advanced scenarios, so my suggestion would be to call them out as such.